api/definitions.yaml

```diff
@@ -336,6 +338,11 @@ IsInboxEnabled:
   description: |-
     True if the recipient of a message wants to store its content for
     later retrieval.
+TosVersion:
+  type: number
+  default: 0
```

Comment:
don't put a default, a missing value means the user has not accepted any Tos yet